server/src/main/java/dev/slimevr/tracking/trackers/TrackerResetsHandler.kt

```diff
@@ -96,6 +96,16 @@ class TrackerResetsHandler(val tracker: Tracker) {
 		return adjustToIdentity(rotation)
 	}
 
+	/**
+	 * Takes a rotation and adjusts it to mounting
+	 */
+	fun getMountingAdjustedRotationFrom(rotation: Quaternion): Quaternion {
+		if (mountRotFix == Quaternion.IDENTITY) {
+			return rotation * mountingOrientation
+		}
+		return rotation * mountRotFix
```

Comment:
Mounting orientation needs to be applied before mounting fix
```suggestion
		if (mountRotFix == Quaternion.IDENTITY) {
			return rotation * mountingOrientation
		}
		return rotation * mountingOrientation * mountRotFix
```

Reply:
i would prefer just removing the if

Reply:
Here's if we don't want the if
```suggestion
		return rotation * mountingOrientation * mountRotFix
```

Reply:
I tested this and it seemed to be the same either way. I believe that that was because I have default mounting on all my trackers so this is very likely right, I will test again soon anyways

Reply:
tested it best I can tell it is working